lua/ide/workspaces/workspace_controller.lua

```diff
@@ -249,7 +249,7 @@ function WorkspaceController.new(config)
         local function init_callback()
             -- do not assign if we are git tool.
             local buf_name = vim.api.nvim_buf_get_name(0)
-            if vim.fn.match(buf_name, ".git/") > -1 then
+            if vim.fn.match(buf_name, "\\.git/") > -1 then
```

Comment:
why this ?

Reply:
It’s from my other PR, I needed it locally. If we merge the other one first this will be eliminated from the diff.

Reply:
Huh? You have no other open pr right now

Reply:
Oh, sorry about that, lol I think I submitted the PR to my own fork by accident, one sec 🙈

Reply:
no worries haha. was confused.